src/main/java/com/bandwidth/iris/sdk/model/TelephoneNumberDetails.java

```diff
@@ -1,79 +0,0 @@
-package com.bandwidth.iris.sdk.model;
-
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
-
-/**
- * Created by sbarstow on 11/20/14.
- */
-@XmlRootElement(name = "TelephoneNumberDetails")
-@XmlAccessorType(XmlAccessType.FIELD)
-public class TelephoneNumberDetails extends TelephoneNumberDetail {
```

Comment:
Not sure about the inheritance here. "Details" is a specialization of "Detail"? I understand that the xml tag "TelphoneNumberDetails" containing detail related to a single telephone number is rather misleading, but I don't think this SDK should repeat that mistake. Or is this because we have `<TelephoneNumber>`, `<TelephoneNumberDetail>` AND `<TelephoneNumberDetails>`? Each represents a single telephone number. Sad.